cfscrape/__init__.py

```diff
@@ -246,22 +246,23 @@ def solve_cf_challenge(self, resp, **original_kwargs):
 
     def solve_challenge(self, body, domain):
         try:
-            all_scripts = re.findall(r'\<script type\=\"text\/javascript\"\>\n(.*?)\<\/script\>',body, flags=re.S)
-            javascript = next(filter(lambda w: "jschl-answer" in w,all_scripts)) #find the script tag which would have obfuscated js
+            javascript = re.search(r'\<script type\=\"text\/javascript\"\>\n(.*?)\<\/script\>',body, flags=re.S).group(1) # find javascript
+
             challenge, ms = re.search(
                 r"setTimeout\(function\(\){\s*(var "
-                r"s,t,o,p,b,r,e,a,k,i,n,g,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"
-                r"(?:[^{<>]*},\s*(\d{4,}))?",
+                r"s,t,o,p, b,r,e,a,k,i,n,g,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"
```

Comment:
This PR only works for me if I add `,cpo` into this line, like:
`r"s,t,o,p, b,r,e,a,k,i,n,g,cpo,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"`

Reply:
> 
> 
> This PR only works for me if I add `,cpo` into this line, like:
> `r"s,t,o,p, b,r,e,a,k,i,n,g,cpo,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"`

It still don't work with me . Can you test this site ? 
https://tubortaxdeal.com
https://www.aichoai.com

Reply:
weird, the JS on those 2 sites looks really different, like it's not even minified or something.  I wonder if they're using a different version of the cloudflare protection stuff?

Reply:
> This PR only works for me if I add `,cpo` into this line, like:
> `r"s,t,o,p, b,r,e,a,k,i,n,g,cpo,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"`

It was added just recently for the websites I tested.
Thanks !

Reply:
> > This PR only works for me if I add `,cpo` into this line, like:
> > `r"s,t,o,p, b,r,e,a,k,i,n,g,cpo,f.+?\r?\n[\s\S]+?a\.value\s*=.+?)\r?\n"`
> 
> It was added just recently for the websites I tested.
> Thanks !

don't work for https://localbitcoins.net

Reply:
Also doesn't work for http://www.mangago.me/read-manga/hataraku_maousama/